docs/best_practices/ecephys.rst

```diff
@@ -12,15 +12,15 @@ Electrodes
 Location
 ~~~~~~~~
 
-The ``location`` column of the :nwb-schema:ref:`sec-ElectrodeTable` should reflect your best estimate of the recorded
+The ``location`` column of the ``ElectrodeTable`` should reflect your best estimate of the recorded
```

Comment:
Big problem I'm seeing now is the old links to various 'special' tables like Electrodes and Trials have since changed or are no longer available? I've tried everything I can think of but it acts like there isn't a label for https://nwb-schema.readthedocs.io/en/latest/format.html?highlight=electrode#groups-general-extracellular-ephys-electrodes